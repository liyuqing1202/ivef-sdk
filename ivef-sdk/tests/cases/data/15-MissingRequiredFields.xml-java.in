<MSG_Ping xmlns="urn:http://www.ivef.org/XMLSchema/IVEF/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:http://www.ivef.org/XMLSchema/IVEF/1.1
\\projsrvr\isis\develop\ivef-sdk\trunk\doc\ivef1.xsd">
	<Header MsgRefId="String" Version="String"/>
	<Body>
		<Ping/>
	</Body>
</MSG_Ping>

