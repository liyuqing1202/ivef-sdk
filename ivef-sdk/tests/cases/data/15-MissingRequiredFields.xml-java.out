TestApp ready for input
TestApp shutting down
